kategory-docs/docs/docs/typeclasses/functor/README.md

```diff
@@ -48,17 +48,17 @@ Try { "x".toInt() }.map { it * 2 }
 none<Int>().map { it * 2 }
 ```
 
-Kategory allows abstract polymorphic code that operates over the evidence of having an instance of a typeclass available. 
-This enables programs that are not coupled to specific datatype implementations. 
-The technique demonstrated below to write polymorphic code is available for all other [Typeclasses](/docs/typeclasses) beside `Functor`.
+Kategory allows abstract polymorphic code that operates over the evidence of having an instance of a typeclass available.
+This enables programs that are not coupled to specific datatype implementations.
+The technique demonstrated below to write polymorphic code is available for all other [Typeclasses]({{ '/docs/typeclasses' | relative_url }}) beside `Functor`.
```

Comment:
This is the unique point in docs, that make reference to this URL, I create the document with empty content just to not return a 404.

Reply:
We can also just remove the link for now since we have no content for top level sections